### PR TITLE
Basic strip mining process (based on the MineProcess)

### PR DESCRIPTION
**Pull request by XerneraC**
Added a basic strip mining command (using the command "strip") that is based on the standard mining command. It works like the legit mode of the mining process, except it tunnels if it can't see any ores.

- [x] Included no UwU's or OwO's

### CODE REVIEW

**Comment by 0-x-2-2** [comment]
massive iq
image

**Comment by XerneraC** [comment]
I cloned it yesterday, and so I had to redo it

**Comment by StijnSimons** [comment]
*or just rebase 🤯*

**Comment by XerneraC** [comment]
I... didn't think of that...
Sorry about the number of commits.

File: src/main/java/baritone/Baritone.java [hunk 1]
```diff
@@ -75,6 +75,7 @@
 
     private FollowProcess followProcess;
     private MineProcess mineProcess;
+	private StripMineProcess stripMineProcess;
```
Comment (0-x-2-2) [inline src/main/java/baritone/Baritone.java]: wat da fuck

File: src/main/java/baritone/Baritone.java [hunk 2]
```diff
@@ -116,6 +117,7 @@ public synchronized void init() {
         {
             followProcess = new FollowProcess(this);
             mineProcess = new MineProcess(this);
+			stripMineProcess = new StripMineProcess(this);
```
Comment (0-x-2-2) [inline src/main/java/baritone/Baritone.java]: wat da fuck why `                              `

File: src/main/java/baritone/Baritone.java [hunk 3]
```diff
@@ -194,6 +196,11 @@ public MineProcess getMineProcess() {
         return this.mineProcess;
     }
 
+	@Override
+	public StripMineProcess getStripMineProcess() {
+		return this.stripMineProcess;
```
Comment (0-x-2-2) [inline src/main/java/baritone/Baritone.java]: bruh again

**Comment by XerneraC** [comment]
Huh... Neither intellij nor sublime text show these indents...
im6
im5
im4
im3
im2
im1

**Comment by XerneraC** [comment]
I just figured it out... My lines have a tab in front of them, while the other lines use spaces...
I'm gonna fix it real quick

**Comment by LoganDark** [comment]
@XerneraC dude the reason baritone uses spaces is so the indentation doesn't go wacky like that

**Comment by XerneraC** [comment]
> @XerneraC dude the reason baritone uses spaces is so the indentation doesn't go wacky like that

Why would it go whacky if they'd use tabs?

**Comment by LoganDark** [comment]
I personally prefer tabs, but since they use spaces, if you use tabs then if the tab width is not 4 it will look wacky (and github's tab width is 8)

**Comment by XerneraC** [comment]
Yeah I know

**Comment by LoganDark** [comment]
then&nbsp;&nbsp;w&nbsp;h&nbsp;y&nbsp;&nbsp;&nbsp;a&nbsp;s&nbsp;k

**Comment by XerneraC** [comment]
You have a circular definition: They use spaces, so the indentation doesn't go whacky, and tabs make the indentation go whacky because they use spaces.

**Comment by LoganDark** [comment]
@XerneraC I mean the reason you need to use spaces is because you're mixing into a codebase that already uses spaces lol

**Comment by XerneraC** [comment]
That‘s why I changed my tabs to spaces. I would‘ve used spaces from the start, but I didn‘t notice the difference because tabs are 4 spaces in my IDE. I misinterpreted your original comment as saying, that tabs inherently make the indentation go whacky. I‘m sorry about that.

**Comment by LoganDark** [comment]
@XerneraC oh no, I love tabs, just that you shouldn't mix them as indentation. Your IDE should be showing whitespace by the way, so the mix would look something like this:

image

**Comment by XerneraC** [comment]
Yeah...
I usually turn that off because I find it distracting, but I guess I‘m gonna turn it back on.

**Comment by xhochn** [comment]
no progress here? would be an awesome feature ;)

**Comment by ronittaleti** [comment]
Any progress, would love a feature like this.

**Comment by leijurv** [comment]
incredibly cringe copy/paste of mineprocess into a new process instead of calling the existing functions or using the existing class

**Pull request closed without merging**